data/mods/blazemod/blaze_ammo.json

```diff
@@ -1,412 +1,386 @@
+    "description": "A large spear carved from wood.  This was fashioned specifically to be fired from a compatible weapon; thus unsuitable for melee combat.",
+    "weight": 1455,
+    "volume": 10,
+    "price": 1000,
+    "bashing": 5,
+    "material": "wood",
+    "symbol": "=",
+    "color": "brown",
+    "count": 10,
+    "ammo_type": "harpoon",
+    "damage": 40,
+    "pierce": 10,
+    "effects": [ "RECOVER_3", "NEVER_MISFIRES" ]
+  },
+  {
+    "id": "hbolt_boom",
+    "type": "AMMO",
+    "name": "nova bolt",
+    "description": "An IED composed of a sealed container filled with a payload material affixed to the end of a large ballista bolt.  Intended to be fired from a specialized launcher.  Carries a substantial explosive payload.",
+    "weight": 3816,
+    "volume": 1,
+    "price": 1000,
+    "bashing": 8,
+    "material": "iron",
+    "symbol": "*",
+    "color": "red",
+    "count": 1,
+    "stack_size": 1,
+    "ammo_type": "hbolt",
+    "damage": 14,
+    "effects": [ "NEVER_MISFIRES", "EXPLOSIVE_BIG", "COOKOFF" ],
+    "flags": [ "" ]
+  },
+  {
+    "id": "hbolt_boom2",
+    "type": "AMMO",
+    "name": "supernova bolt",
+    "description": "An IED composed of a sealed container filled with a payload material affixed to the end of a large ballista bolt.  Intended to be fired from a specialized launcher.  Carries a very substantial explosive payload.",
+    "weight": 3816,
+    "volume": 1,
+    "price": 1000,
+    "bashing": 8,
+    "material": "iron",
+    "symbol": "*",
+    "color": "red",
+    "count": 1,
+    "stack_size": 1,
+    "ammo_type": "hbolt",
+    "damage": 14,
+    "effects": [ "NEVER_MISFIRES", "EXPLOSIVE_HUGE", "COOKOFF" ],
+    "flags": [ "" ]
+  },
+  {
+    "id": "hbolt_fire",
+    "type": "AMMO",
+    "name": "searing bolt",
+    "description": "An IED composed of a container filled with a payload material affixed to the end of a large ballista bolt.  Intended to be fired from a specialized launcher.  This one is designed to douse a small area with deadly flames.",
+    "weight": 3816,
+    "volume": 1,
+    "price": 600,
+    "bashing": 8,
+    "material": "iron",
+    "symbol": "*",
+    "color": "red",
+    "count": 1,
+    "stack_size": 1,
+    "ammo_type": "hbolt",
+    "damage": 14,
+    "effects": [ "FLAME", "NEVER_MISFIRES", "COOKOFF" ],
+    "flags": [ "" ]
+  },
+  {
+    "id": "hbolt_fire2",
+    "type": "AMMO",
+    "name": "napalm bolt",
+    "description": "An IED composed of a container filled with a payload material affixed to the end of a large ballista bolt.  Intended to be fired from a specialized launcher.  This one is designed to douse a large area with deadly flames.",
+    "weight": 6816,
+    "volume": 1,
+    "price": 2200,
+    "bashing": 8,
+    "material": "iron",
+    "symbol": "*",
+    "color": "red",
+    "count": 1,
+    "stack_size": 1,
+    "ammo_type": "hbolt",
+    "damage": 14,
+    "effects": [ "NAPALM", "NEVER_MISFIRES", "COOKOFF" ],
+    "flags": [ "" ]
+  },
```

Comment:
This entire line of items is a giant block of horrible copy+paste. Would benefit from some "similar json detector" system that would replace sets of definitions with `copy-from` entry.
Extending linter to do that would probably be too much work (it's too specialized), but I could write a custom script just to do that for this PR.